safenode/src/protocol/mod.rs

```diff
@@ -23,6 +23,8 @@ pub mod error;
 pub mod fees;
 /// Token transfers.
 pub mod offline_transfers;
+/// Token transfers.
```

Comment:
better to expand the comments of `offline_transfers` and `online_transfers` to explain little bit what's the different is?

Reply:
These are merged in the follow up PR.

> NB 1: Some duplication of types in the online transfers mod. This will be improved upon in the followup PRs.

Reply:
These are merged into one in the follow up PR.

> NB 1: Some duplication of types in the online transfers mod. This will be improved upon in the followup PRs.